noxfile.py

```diff
@@ -175,11 +175,25 @@ def fragment_alternative_templates(session):
     fragment(session, use_ads_templates=True)
 
 
+def _add_python_settings(tmp_dir, python_settings):
+    return f"""
+import yaml
+from pathlib import Path
+temp_file_path = Path(f"{tmp_dir}/showcase_v1beta1.yaml")
+with temp_file_path.open('r') as file:
+    data = yaml.safe_load(file)
+    data['publishing']['library_settings'] = {python_settings}
+
+with temp_file_path.open('w') as file:
+    yaml.safe_dump(data, file, default_flow_style=False, sort_keys=False)
+"""
+
 @contextmanager
 def showcase_library(
     session, templates="DEFAULT", other_opts: typing.Iterable[str] = (),
     include_service_yaml=True,
     retry_config=True,
+    rest_async_io_enabled=False
```

Comment:
This line should also have a TODO to remove the param once async REST is GA?

Reply:
addressed.